### ISSUE TITLE
Using the plugin with different devices 

### ISSUE THREAD

**Issue by rickdps** [NathanaelA/nativescript-orientation#33]
Hi, Nathanael. First of all, thank you very much for this plugin. It has helped me a lot. 

My questions is: 

Is it possible to use the plugin like this: 
.landscape .tablet .classname {

}

I have tried it but it did not work and just wanted to check if it's because it doesn't work or if I'm doing something wrong. 

Thank you.

**Comment by NathanaelA** [comment]
Yes, you can combine css rules; it really depends on what assigned the `.tablet` part of the rule.   If you use this with my NS-platform-css you can then you can use the .tablet and the .landscape together easier since the ns-platform-css will defined .tablet for you.

**Comment by rickdps** [comment]
Ok. I have them both, nativescript-platform-css and nativescript-orientation plugins. Could you please give me an example of how to implement it. As I said, '.landscape .tablet .classname {}' is not working.

**Comment by YaakovDavid** [comment]
Each plugin separately works but together I can't get it to work for (CSS), I tried ```.tablet .landscape``` and the reverse ```.landscape .tablet```. If there is an example of how to combine the two it would be appreciated.
